.github/workflows/electron-exe-build.yml

```
@@ -0,0 +1,261 @@
+  workflow_dispatch:
+  # push:
+
+jobs:
+  build-windows:
+    runs-on: windows-latest
+    outputs:
+      upload_url_base64: ${{ steps.b64encode_upload_url.outputs.upload_url_base64 }}
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)